src/test/java/com/mathworks/ci/RunMatlabBuildBuilderTest.java

```diff
@@ -129,6 +129,7 @@ public void verifyMATLABlaunchedWithDefaultArguments() throws Exception {
         this.buildWrapper.setMatlabBuildWrapperContent(new MatlabBuildWrapperContent(Message.getValue("matlab.custom.location"), getMatlabroot("R2017a")));
         project.getBuildWrappersList().add(this.buildWrapper);
         scriptBuilder.setTasks("");
+        scriptBuilder.setBuildOptions(new BuildOptions(""));
```

Comment:
One last nitpick: Remove all of these statements from the tests that aren't testing build options. The null handling should mean that they're not necessary any more.

Reply:
Understood. I have updated the tests!